faq/migration-tidb-faq.md

```diff
@@ -50,9 +50,14 @@ DB2、Oracle 到 TiDB 数据迁移（增量+全量），通常做法有：
 
 - 也可以选择增大 tidb 的单个事物语句数量限制，不过这个会导致内存上涨。
 
-### Dumpling 导出大表时引发上游数据库报错“磁盘空间不足”
+### Dumpling 导出时引发上游数据库 OOM 或报错“磁盘空间不足”
 
-该问题是由于数据库主键分布不均匀，Dumpling 划分导出子范围时出现过大的子范围引起的。请尝试分配更大的磁盘空间，或者联系 [AskTUG 社区专家](https://asktug.com/) 获取实验版本的 Dumpling。
+该问题可能有如下原因：
+
+- 数据库主键分布不均匀，例如启用了 [SHARD_ROW_ID_BITS](/shard-row-id-bits.md)
+- 上游数据库为 TiDB，导出表是分区表
+
+在上述情况下，Dumpling 划分导出子范围时，会划分出过大的子范围，从而向上游发送结果过大的查询。请联系 [AskTUG 社区专家](https://asktug.com/) 获取实验版本的 Dumpling。
```

Comment:
Any alternative channel for AskTUG in the English version? Slack?

Reply:
Slack is OK

Reply:
Do you have the link for Slack?

Reply:
https://slack.tidb.io/invite?team=tidb-community&channel=sig-migrate&ref=github_sig